repositories/user.js

```diff
@@ -10,6 +10,25 @@ async function getUserByEmail(email) {
     return await User.findOne({ email: email })
 }
 
+async function createUser(email, hashedPassword, role) {
+    if (!email) {
```

Comment:
I think you are already check this in service. And also we need to care this in service not in repository